src/main/resources/db/migration/V5__Filling _dummy_data_into_DB.sql

```diff
@@ -0,0 +1,167 @@
+  (2, 3, 'User', 'user@user.user', '$2a$10$DDb4kmE5su.kXx7ku11slubKGBoQfEaCdSxIGokN7QUa46ync48o2');
+
+--   ----------------------------------------------------------------------------------------------
+-- Fill users_roles table with user PASSWORD = admin
+--   ----------------------------------------------------------------------------------------------
+INSERT INTO startup_platform.users_roles (user_id, role_id) VALUES
+  (1, 2),
+  (2, 1);
+
+--   ----------------------------------------------------------------------------------------------
+--  Fill categories table
+--   ----------------------------------------------------------------------------------------------
+INSERT INTO startup_platform.categories (id, name) VALUES
+  (1, 'CATEGORY 1'),
+  (2, 'CATEGORY 2'),
+  (3, 'CATEGORY 3');
+
+--   ----------------------------------------------------------------------------------------------
+--  Fill investment_interest table
+--   ----------------------------------------------------------------------------------------------
+INSERT INTO startup_platform.investment_interests (id, category_id, sum, description, goal, user_id) VALUES
+  (1, 1, 500, 'Description for FIRST InvInterest', 'Goal for FIRST InvInterest', 1),
+  (2, 2, 500, 'Description for SECOND InvInterest', 'Goal for SECOND InvInterest', 2),
+  (3, 3, 500, 'Description for THIRD InvInterest', 'Goal for THIRD InvInterest', 1);
+
+--   ----------------------------------------------------------------------------------------------
+--  Fill projects table
+--   ----------------------------------------------------------------------------------------------
+INSERT INTO startup_platform.projects (id, name, owner_id, category_id, cost, date_added, date_start, min_invest, description, goal, stage)
+VALUES
+  (1, 'Raccoon.World - gadgets for hands', 1, 1, 400000, '20.05.2017 14:30', '20.05.2017 15:30', 50000, 'Raccoon.World is a company that creates gadgets for hands, based on spatialization, tracking, reproduction of fine motor skills and tactile perception.
+Raccoon.world is an international company incorporated in Delaware, USA with R&D Center in Ukraine.', 'goal', 'start'),
+  (2, 'Integrating Virtuality into Reality, Game', 1, 1, 150000, '20.05.2017 14:30', '20.05.2017 15:30', 50000,
+      'Mobile MMO RPG (midcore), with Step-by-step combat system, and focus on multiplayer real-time PvP.', 'goal',
+      'start'),
+  (3, 'Wood Pellets Storage', 1, 2, '1800000', '20.05.2017 14:30', '20.05.2017 15:30', 60000, 'European winter storage
+All math and figures are taken from personal experience obtained during last year
+Pellets sales market- Europe', 'goal', 'start'),
+  (4, 'Virtual Insurance Agents (chatbots) for insurers', 1, 2, 400000, '20.05.2017 14:30', '20.05.2017 15:30', 20000,
+      'We are building Virtual Insurance Agents (chatbots) to help insurers to engage their customers via Facebook Messengers.',
+      'goal', 'middle'),
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Define a constant instead of duplicating this literal 'middle' 4 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.com/coding_rules#rule_key=plsql%3AS1192)